package-lock.json

```diff
@@ -85,13 +85,40 @@
         "apollo-env": "^0.6.5"
       }
     },
-    "@apollographql/graphql-playground-html": {
-      "version": "1.6.26",
-      "resolved": "https://registry.npmjs.org/@apollographql/graphql-playground-html/-/graphql-playground-html-1.6.26.tgz",
-      "integrity": "sha512-XAwXOIab51QyhBxnxySdK3nuMEUohhDsHQ5Rbco/V1vjlP75zZ0ZLHD9dTpXTN8uxKxopb2lUvJTq+M4g2Q0HQ==",
+    "@apollographql/graphql-upload-8-fork": {
```

Comment:
Can you revert the this file?

Reply:
Yeah I did it, maybe that's why the CI build was failing because it couldn't find `graphql` module